src/lib.rs

```diff
@@ -523,8 +523,6 @@ pub trait Rng: RngCore {
 
     /// Return a bool with a probability `p` of being true.
     ///
-    /// This is a wrapper around [`distributions::Bernoulli`].
```

Comment:
Please at least mention this in the doc; same with `gen_ratio` below.

Reply:
It's a comment, so I'll add this back if really desired. However I don't think it's a good idea to promise a particular implementation strategy. That just seems like complicate future arguments regarding if changing the implementation is a breaking change or not.

If we want a pointer to `Bernoulli`, maybe do what we do for `gen_range` and say: See also the [`Bernoulli`] distribution type which may be faster if sampling with the same probability repeatedly.

Reply:
Sure, just a pointer to `Bernoulli` is sufficient. You're right that we don't need to specify this constraint in the doc, though I don't see it as a big deal.